servers/src/test/java/tachyon/master/block/meta/MasterBlockInfoTests.java

```diff
@@ -1,87 +0,0 @@
-/*
```

Comment:
because there is an identical `MasterBlockInfoTest.java` file already ... unless I am missing something